minifi/pom.xml

```diff
@@ -833,7 +833,7 @@ limitations under the License.
             <dependency>
                 <groupId>com.h2database</groupId>
                 <artifactId>h2</artifactId>
-                <version>1.4.199</version>
+                <version>2.1.210</version>
```

Comment:
Seems like we have a few places that directly use the version rather than referencing `${h2.version}`, I'm guessing the idea being that they may evolve independently, but would that ever really happen in practice? Seems like we'd always probably upgrade all usage at the same time since all in the same repo now.